src/test/java/com/wikia/webdriver/common/WindowSize.java

```diff
@@ -0,0 +1,8 @@
+package com.wikia.webdriver.common;
+
+import org.openqa.selenium.Dimension;
+
+public class WindowSize {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)